R/idr0021 - Segmentation.ipynb

```diff
@@ -0,0 +1,702 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Segmentation using the idr0021 dataset\n",
+    "\n",
+    "This notebook uses the idr0021 dataset (original: [idr0021-lawo-pericentriolarmaterial/experimentA](http://idr.openmicroscopy.org/webclient/?show=project-51)) and tries to reproduce some of the analysis published in ['Subdiffraction imaging of centrosomes reveals higher-order organizational features of pericentriolar material'](https://doi.org/10.1038/ncb2591). In particular we're trying to get a figure similar to [Figure 1](https://www.nature.com/articles/ncb2591/figures/1) of the article. \n",
+    "\n",
+    "Each user has a partial copy of the dataset ('Lab1' / YOUR USER / Project: 'idr0021'). Some of the images are also part of the dataset 'R-dataset' which you can use for practicing the segmentation and annotation steps."
```

Comment:
Instead of ``('Lab1' / YOUR USER / Project: 'idr0021')`` just ``...inside the Project "idr0021"

Reply:
But actually, this is so very much outreach specific... Maybe we would like to put such sentence rather in the workflow .pdf than here.

Reply:
Agree better to put in the pdf